packages/core/package.json

```diff
@@ -75,6 +75,7 @@
     "@mui/material": "5.14.5",
     "@mui/styled-engine-sc": "5.12.0",
     "@mui/styles": "5.14.5",
+    "@mui/x-charts": "7.3.2",
```

Comment:
GItHub is warning that '@mui/x-charts' should be listed in the project's dependencies, not devDependencies